src/core/execute.c

```diff
@@ -3148,6 +3148,16 @@ static int exec_child(
                         return log_unit_error_errno(unit, errno, "Failed to set up CPU affinity: %m");
                 }
 
+        if (mpol_is_valid(numa_policy_get_type(&context->numa_policy))) {
+                r = apply_numa_policy(&context->numa_policy);
+                if (r == -EOPNOTSUPP)
+                        log_unit_debug_errno(unit, SYNTHETIC_ERRNO(r), "NUMA support not available, ignoring.");
```

Comment:
the error is not synthetic here (we are not generating it here, but received it already from the previous call). Please drop the SYNTHETIC_ERRNO() wrapper here, just use `r` directly.

Reply:
Hmm, but `apply_numa_policy()` does produce value synthetically if we detect that syscall is not supported.

Reply:
SYNTHETIC_ERRNO() just means "generated here", nothing more. Anything which is passed down from a called function should be just logged normally.

Reply:
Ahh, ok. I will send followup then.